真寻长时间运行会导致无法处理消息

### 操作系统

Windows

### Python 版本

3.10

### zhenxun_bot 版本

v0.2.4-c7c759f

### 适配器

onebot v11 2.3.3

### 协议端

napcat v4.2.28 lts

### 描述问题

实测为长时间运行导致的消息处理均被skip

### 复现步骤

长时间连续运行bot
二编：
查看上次日志后发现具体时间是6h
三编：
这次我Bot刚开机40min就触发了，这个太随机了

### 期望的结果

_No response_

### 截图或日志（请勿包含敏感信息如密码、令牌等）
```log
12-15 09:21:36 [INFO] zhenxun | 加载配置完成，共加载 44 个配置组及对应 128 个配置项
12-15 09:21:36 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_apscheduler"
12-15 09:21:36 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_alconna:uniseg" from "nonebot_plugin_alconna.uniseg"
12-15 09:21:37 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_alconna"
12-15 09:21:37 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_userinfo"
12-15 09:21:37 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_htmlrender"
12-15 09:21:38 [SUCCESS] nonebot | Succeeded to load plugin "superuser:bot_manage" from "zhenxun.builtin_plugins.superuser.bot_manage"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:request_manage" from "zhenxun.builtin_plugins.superuser.request_manage"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:set_admin" from "zhenxun.builtin_plugins.superuser.set_admin"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:broadcast" from "zhenxun.builtin_plugins.superuser.broadcast"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:group_manage" from "zhenxun.builtin_plugins.superuser.group_manage"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:fg_manage" from "zhenxun.builtin_plugins.superuser.fg_manage"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:reload_setting" from "zhenxun.builtin_plugins.superuser.reload_setting"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:super_help" from "zhenxun.builtin_plugins.superuser.super_help"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:update_fg_info" from "zhenxun.builtin_plugins.superuser.update_fg_info"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:clear_data" from "zhenxun.builtin_plugins.superuser.clear_data"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser:exec_sql" from "zhenxun.builtin_plugins.superuser.exec_sql"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "superuser" from "zhenxun.builtin_plugins.superuser"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "restart" from "zhenxun.builtin_plugins.restart"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "statistics:statistics_handle" from "zhenxun.builtin_plugins.statistics.statistics_handle"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "statistics:statistics_hook" from "zhenxun.builtin_plugins.statistics.statistics_hook"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "statistics" from "zhenxun.builtin_plugins.statistics"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "catchphrase" from "zhenxun.builtin_plugins.catchphrase"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "plugin_store" from "zhenxun.builtin_plugins.plugin_store"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "scripts" from "zhenxun.builtin_plugins.scripts"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "scheduler:chat_check" from "zhenxun.builtin_plugins.scheduler.chat_check"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "scheduler:auto_update_group" from "zhenxun.builtin_plugins.scheduler.auto_update_group"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "scheduler:auto_backup" from "zhenxun.builtin_plugins.scheduler.auto_backup"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "scheduler:morning" from "zhenxun.builtin_plugins.scheduler.morning"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "scheduler" from "zhenxun.builtin_plugins.scheduler"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "platform:exception" from "zhenxun.builtin_plugins.platform.qq.exception"
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "platform:group_handle" from "zhenxun.builtin_plugins.platform.qq.group_handle"
12-15 09:21:39 [WARNING] zhenxun | 未安装 qq-adapter，无法加载QQ官平台专用插件...
12-15 09:21:39 [SUCCESS] nonebot | Succeeded to load plugin "platform" from "zhenxun.builtin_plugins.platform"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "web_ui" from "zhenxun.builtin_plugins.web_ui"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "init:manager" from "zhenxun.builtin_plugins.init.manager"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "init:init_plugin" from "zhenxun.builtin_plugins.init.init_plugin"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "init:init_config" from "zhenxun.builtin_plugins.init.init_config"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "init" from "zhenxun.builtin_plugins.init"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "nickname" from "zhenxun.builtin_plugins.nickname"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "chat_history:chat_message" from "zhenxun.builtin_plugins.chat_history.chat_message"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "chat_history:chat_message_handle" from "zhenxun.builtin_plugins.chat_history.chat_message_handle"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "chat_history" from "zhenxun.builtin_plugins.chat_history"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "about" from "zhenxun.builtin_plugins.about"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "help_help" from "zhenxun.builtin_plugins.help_help"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "auto_update" from "zhenxun.builtin_plugins.auto_update"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "help" from "zhenxun.builtin_plugins.help"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "sign_in" from "zhenxun.builtin_plugins.sign_in"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "info" from "zhenxun.builtin_plugins.info"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "check" from "zhenxun.builtin_plugins.check"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "shop" from "zhenxun.builtin_plugins.shop"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "admin:plugin_switch" from "zhenxun.builtin_plugins.admin.plugin_switch"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "admin:ban" from "zhenxun.builtin_plugins.admin.ban"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "admin:welcome_message" from "zhenxun.builtin_plugins.admin.welcome_message"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "admin:admin_help" from "zhenxun.builtin_plugins.admin.admin_help"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "admin:group_update" from "zhenxun.builtin_plugins.admin.group_update"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "admin:group_member_update" from "zhenxun.builtin_plugins.admin.group_member_update"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "admin:admin_watch" from "zhenxun.builtin_plugins.admin.admin_watch"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "admin" from "zhenxun.builtin_plugins.admin"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "record_request" from "zhenxun.builtin_plugins.record_request"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "withdraw" from "zhenxun.builtin_plugins.withdraw"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "hooks:auth_hook" from "zhenxun.builtin_plugins.hooks.auth_hook"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "hooks:chkdsk_hook" from "zhenxun.builtin_plugins.hooks.chkdsk_hook"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "hooks:ban_hook" from "zhenxun.builtin_plugins.hooks.ban_hook"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "hooks:call_hook" from "zhenxun.builtin_plugins.hooks.call_hook"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "hooks:withdraw_hook" from "zhenxun.builtin_plugins.hooks.withdraw_hook"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "hooks" from "zhenxun.builtin_plugins.hooks"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "mc_check" from "zhenxun.plugins.mc_check"
12-15 09:21:40 [SUCCESS] nonebot | Succeeded to load plugin "roll" from "zhenxun.plugins.roll"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "comments_163" from "zhenxun.plugins.comments_163"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "dialogue" from "zhenxun.plugins.dialogue"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "hello" from "zhenxun.plugins.hello"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "image_management:send_image" from "zhenxun.plugins.image_management.send_image"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "image_management:delete_image" from "zhenxun.plugins.image_management.delete_image"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "image_management:upload_image" from "zhenxun.plugins.image_management.upload_image"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "image_management:move_image" from "zhenxun.plugins.image_management.move_image"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "image_management:image_management_log" from "zhenxun.plugins.image_management.image_management_log"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "image_management" from "zhenxun.plugins.image_management"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "send_setu_:update_setu" from "zhenxun.plugins.send_setu_.update_setu"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "send_setu_:send_setu" from "zhenxun.plugins.send_setu_.send_setu"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "send_setu_" from "zhenxun.plugins.send_setu_"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "wbtop" from "zhenxun.plugins.wbtop"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "search_anime" from "zhenxun.plugins.search_anime"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "pa" from "zhenxun.plugins.pa"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "parse_bilibili" from "zhenxun.plugins.parse_bilibili"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "russian" from "zhenxun.plugins.russian"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "search_image" from "zhenxun.plugins.search_image"
12-15 09:21:41 [SUCCESS] nonebot | Succeeded to load plugin "bilibili_sub" from "zhenxun.plugins.bilibili_sub"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "word_bank:word_handle" from "zhenxun.plugins.word_bank.word_handle"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "word_bank:message_handle" from "zhenxun.plugins.word_bank.message_handle"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "word_bank:read_word_bank" from "zhenxun.plugins.word_bank.read_word_bank"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "word_bank:exception" from "zhenxun.plugins.word_bank.exception"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "word_bank" from "zhenxun.plugins.word_bank"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "poke" from "zhenxun.plugins.poke"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "jitang" from "zhenxun.plugins.jitang"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "what_role" from "zhenxun.plugins.what_role"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "black_word:black_word" from "zhenxun.plugins.black_word.black_word"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "black_word:data_source" from "zhenxun.plugins.black_word.data_source"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "black_word:model" from "zhenxun.plugins.black_word.model"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "black_word:utils" from "zhenxun.plugins.black_word.utils"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "black_word:black_watch" from "zhenxun.plugins.black_word.black_watch"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "black_word" from "zhenxun.plugins.black_word"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "group_welcome_msg" from "zhenxun.plugins.group_welcome_msg"
12-15 09:21:42 [SUCCESS] nonebot | Succeeded to load plugin "music" from "zhenxun.plugins.music"
12-15 09:21:43 [SUCCESS] nonebot | Succeeded to load plugin "word_clouds" from "zhenxun.plugins.word_clouds"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "pix_api:pix_set" from "zhenxun.plugins.pix_api.pix_set"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "pix_api:token" from "zhenxun.plugins.pix_api.token"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "pix_api:pix_keyword" from "zhenxun.plugins.pix_api.pix_keyword"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "pix_api:pix" from "zhenxun.plugins.pix_api.pix"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "pix_api:pix_star" from "zhenxun.plugins.pix_api.pix_star"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "pix_api:pix_info" from "zhenxun.plugins.pix_api.pix_info"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "pix_api:pix_tag" from "zhenxun.plugins.pix_api.pix_tag"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "pix_api" from "zhenxun.plugins.pix_api"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "gold_redbag" from "zhenxun.plugins.gold_redbag"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "what_anime" from "zhenxun.plugins.what_anime"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_kawaii_status" from "nonebot_plugins.nonebot_plugin_kawaii_status"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_moyu" from "nonebot_plugins.nonebot_plugin_moyu"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_localstore"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_datastore"
12-15 09:21:44 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_bottle" from "nonebot_plugins.nonebot_plugin_bottle"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "nonebot-plugin-with-ai-agents-main" from "nonebot_plugins.nonebot-plugin-with-ai-agents-main"
12-15 09:21:45 [INFO] nonebot_plugin_mute | 禁言时间配置:[1, 5, 10, 30]
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_mute" from "nonebot_plugins.nonebot_plugin_mute"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_homonumber-main" from "nonebot_plugins.nonebot_plugin_homonumber-main"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_easy_group_manager" from "nonebot_plugins.nonebot_plugin_easy_group_manager"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "nonebot_plugin_emojimix" from "nonebot_plugins.nonebot_plugin_emojimix"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "zhenxun_plugin_code-main" from "load_plugins.zhenxun_plugin_code-main"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "crazy_thursday" from "load_plugins.crazy_thursday"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "todaybefore" from "load_plugins.todaybefore"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "logo" from "load_plugins.logo"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "cp" from "load_plugins.cp"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "text" from "load_plugins.text"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "zhenxun_plugin_meiriyiju-main" from "load_plugins.zhenxun_plugin_meiriyiju-main"
12-15 09:21:45 [SUCCESS] nonebot | Succeeded to load plugin "what2eat" from "load_plugins.what2eat"
12-15 09:21:45 [SUCCESS] nonebot | Running NoneBot...
12-15 09:21:45 [INFO] uvicorn | Started server process [532]
12-15 09:21:45 [INFO] uvicorn | Waiting for application startup.
12-15 09:21:45 [INFO] zhenxun | Database loaded successfully!
12-15 09:21:45 [INFO] nonebot_plugin_apscheduler | Scheduler Started
12-15 09:21:46 [INFO] nonebot_plugin_htmlrender | 使用 chromium 启动
12-15 09:21:46 [INFO] nonebot_plugin_htmlrender | Browser Started.
12-15 09:21:47 [INFO] zhenxun | CMD[Web UI] API启动成功
12-15 09:21:47 [INFO] zhenxun | 已经加载 8 个插件限制.
12-15 09:22:01 [INFO] nonebot_plugin_datastore | 数据库初始化完成
12-15 09:22:01 [INFO] uvicorn | Application startup complete.
12-15 09:22:01 [INFO] uvicorn | Uvicorn running on http://127.0.0.1:8080 (Press CTRL+C to quit)
12-15 09:22:04 [INFO] uvicorn | ('127.0.0.1', 49639) - "WebSocket /onebot/v11/ws" [accepted]
12-15 09:22:04 [INFO] nonebot | OneBot V11 | Bot BOT_QQ号 connected
12-15 09:22:04 [INFO] websockets | connection open
12-15 09:24:06 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.private.friend]: Message 328478818 from QQ号 '签到'
12-15 09:24:06 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:24:06 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:24:06 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running complete
12-15 09:24:06 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:24:06 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:24:07 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=90)
12-15 09:24:07 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running complete
12-15 09:24:07 [INFO] zhenxun | Adapter[OneBot V11] 平台[QQClient] 用户[QQ号] CMD[签到] 签到成功. score: 215.69 (+0.29).获取金币/道具: 96
12-15 09:24:09 [INFO] zhenxun | Adapter[OneBot V11] 平台[QQClient] 用户[QQ号] CMD[签到] 签到成功
12-15 09:24:10 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=90) running complete
12-15 09:24:12 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.private.friend]: Message 807241555 from QQ号 '[reply:id=1920580553]漫画上色'
12-15 09:24:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:24:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:24:12 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:24:12 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running complete
12-15 09:24:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:24:12 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running complete
12-15 09:24:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.help_help, lineno=32)
12-15 09:24:12 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.help_help, lineno=32) running complete
12-15 09:24:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=nonebot_plugins.nonebot-plugin-with-ai-agents-main.handler, lineno=9)
12-15 09:24:12 [INFO] nonebot-plugin-with-ai-agents-main | 智脑分类结果：[7]
12-15 09:24:12 [INFO] nonebot-plugin-with-ai-agents-main | assemble_prompt
12-15 09:24:20 [INFO] nonebot | Matcher(type='message', module=nonebot_plugins.nonebot-plugin-with-ai-agents-main.handler, lineno=9) running complete
12-15 09:33:36 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 208674813 from 2445507615@[群:704262408] '早安'
12-15 09:33:36 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:33:36 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:33:36 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:33:37 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 927042796 from 2445507615@[群:704262408] '签到'
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running complete
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:33:37 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1147709841 from 2445507615@[群:704262408] '赞我'
12-15 09:33:37 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 32058104 from 2445507615@[群:704262408] '娶群友'
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:33:37 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1973433908 from 2188693166@[群:704262408] '[reply:id=208674813]早安成功！你是今天第0个起床的'
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running is cancelled
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running complete
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running is cancelled
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:33:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running complete
12-15 09:33:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:33:38 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running complete
12-15 09:33:38 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=90)
12-15 09:33:38 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:33:38 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:33:38 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running is cancelled
12-15 09:33:38 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running is cancelled
12-15 09:33:38 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running is cancelled
12-15 09:33:38 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running is cancelled
12-15 09:33:38 [INFO] zhenxun | Adapter[OneBot V11] 平台[QQClient] 群聊[704262408] 用户[2445507615] CMD[签到] 签到成功. score: 170.11 (+0.94).获取金币/道具: 49
12-15 09:33:39 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1064020080 from 2188693166@[群:704262408] '[reply:id=1147709841]给你赞了10下哦，记得回我~'
12-15 09:33:39 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:33:39 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:33:39 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:33:39 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running is cancelled
12-15 09:33:39 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:33:39 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running is cancelled
12-15 09:33:40 [INFO] zhenxun | Adapter[OneBot V11] 平台[QQClient] 群聊[704262408] 用户[2445507615] CMD[签到] 签到成功
12-15 09:33:42 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 546964699 from 2188693166@[群:704262408] '[at:qq=2445507615]今天你的群老婆是[image:summary=,file=000001464e61704361744f6e65426f747c4d736746696c657c327c3730343236323...,sub_type=0,file_id=000001464e61704361744f6e65426f747c4d736746696c657c327c3730343236323...,url=https://multimedia.nt.qq.com.cn/download?appid=1407&fileid=EhR56xIJ...,file_size=88173,file_unique=b843166a45979269911e8d2bc91a11b1]\n[云冒险](1741856095)哒\n当前你们好感度为6'
12-15 09:33:42 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:33:42 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:33:42 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:33:42 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running is cancelled
12-15 09:33:42 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:33:43 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=90) running complete
12-15 09:33:43 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32) running is cancelled
12-15 09:33:48 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1253270740 from 2188693166@[群:704262408] '[image:summary=,file=000001464e61704361744f6e65426f747c4d736746696c657c327c3730343236323...,sub_type=0,file_id=000001464e61704361744f6e65426f747c4d736746696c657c327c3730343236323...,url=https://multimedia.nt.qq.com.cn/download?appid=1407&fileid=EhSWoX46...,file_size=155183,file_unique=fe08078c9647f67dc53a75d2ccaf4f5c]'
12-15 09:33:48 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:33:48 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
12-15 09:33:48 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53)
12-15 09:33:48 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=53) running is cancelled
12-15 09:33:48 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.black_word.black_watch, lineno=32)
12-15 09:34:33 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 2025059665 from 1700915198@[群:891618185] '[image:summary=[动画表情],file=000001464e61704361744f6e65426f747c4d736746696c657c327c3839313631383...,sub_type=1,file_id=000001464e61704361744f6e65426f747c4d736746696c657c327c3839313631383...,url=https://multimedia.nt.qq.com.cn/download?appid=1407&fileid=EhQO3d4K...,file_size=686485,file_unique=6d8dbcd2503239d8e5f137b52cedee78]'
12-15 09:34:33 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:35:39 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:35:39 CST)" skipped: maximum number of running instances reached (1)
12-15 09:35:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:35:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:36:35 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1422653723 from 1700915198@[群:891618185] '逆天残局，对面全轮椅😰😰😰[image:summary=,file=000001464e61704361744f6e65426f747c4d736746696c657c327c3839313631383...,sub_type=0,file_id=000001464e61704361744f6e65426f747c4d736746696c657c327c3839313631383...,url=https://multimedia.nt.qq.com.cn/download?appid=1407&fileid=EhSQJr0X...,file_size=67411,file_unique=f4ed6ae7c0a558a479eae5d503b30f61]'
12-15 09:36:35 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:36:39 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:36:39 CST)" skipped: maximum number of running instances reached (1)
12-15 09:36:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:36:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:36:56 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 170253978 from 1700915198@[群:891618185] '我们191真是太有实力了👍👍👍'
12-15 09:36:56 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:37:39 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:37:39 CST)" skipped: maximum number of running instances reached (1)
12-15 09:37:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:37:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:38:39 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:38:39 CST)" skipped: maximum number of running instances reached (1)
12-15 09:38:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:38:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:39:39 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:39:39 CST)" skipped: maximum number of running instances reached (1)
12-15 09:39:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:39:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:40:39 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:40:39 CST)" skipped: maximum number of running instances reached (1)
12-15 09:40:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:40:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:40:50 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1512529516 from 2463613601@[群:704262408] '早安'
12-15 09:40:50 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:40:51 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1675430102 from 2463613601@[群:704262408] '签到'
12-15 09:40:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:40:51 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1265652996 from 2188693166@[群:704262408] '[reply:id=1512529516]早安成功！你是今天第0个起床的'
12-15 09:40:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:40:51 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1982926010 from 2463613601@[群:704262408] '赞我'
12-15 09:40:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:40:52 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1668107502 from 2463613601@[群:704262408] '娶群友'
12-15 09:40:52 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:40:53 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1939700329 from 2188693166@[群:704262408] '[reply:id=1982926010]给你赞了10下哦，记得回我~'
12-15 09:40:53 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:40:53 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1180363526 from 2188693166@[群:704262408] '呜...没娶到，你可以再尝试一次'
12-15 09:40:53 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:40:59 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1188611504 from 2188693166@[群:704262408] '[image:summary=,file=000001464e61704361744f6e65426f747c4d736746696c657c327c3730343236323...,sub_type=0,file_id=000001464e61704361744f6e65426f747c4d736746696c657c327c3730343236323...,url=https://multimedia.nt.qq.com.cn/download?appid=1407&fileid=EhSj04--...,file_size=159966,file_unique=733ec9d48ac001a4bd56c8e3a70f1cdd]'
12-15 09:40:59 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:41:19 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 665448428 from QQ号@[群:704262408] '赞我'
12-15 09:41:19 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:41:21 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.group.normal]: Message 1377017244 from 2188693166@[群:704262408] '[reply:id=665448428]给你赞了10下哦，记得回我~'
12-15 09:41:21 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:41:39 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:41:39 CST)" skipped: maximum number of running instances reached (1)
12-15 09:41:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:41:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:41:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:05:00], next run at: 2024-12-15 09:41:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:42:39 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:42:39 CST)" skipped: maximum number of running instances reached (1)
12-15 09:42:40 [WARNING] apscheduler | Execution of job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:42:40 CST)" skipped: maximum number of running instances reached (1)
12-15 09:42:47 [SUCCESS] nonebot | OneBot V11 BOT_QQ号 | [message.private.friend]: Message 1562254025 from QQ号 '关于'
12-15 09:42:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
12-15 09:43:11 [INFO] uvicorn | Shutting down
12-15 09:43:11 [WARNING] nonebot | OneBot V11 | WebSocket for Bot BOT_QQ号 closed by peer
12-15 09:43:11 [INFO] websockets | connection closed
12-15 09:43:11 [INFO] uvicorn | Waiting for application shutdown.
12-15 09:43:11 [INFO] nonebot_plugin_htmlrender | Browser Stopped.
12-15 09:43:11 [INFO] nonebot_plugin_apscheduler | Scheduler Shutdown
12-15 09:43:11 [ERROR] apscheduler | Job "_ (trigger: interval[0:01:00], next run at: 2024-12-15 09:43:40 CST)" raised an exception
Traceback (most recent call last):
```

今天遇到一次
非长时间，刚启动5分钟，遇到一个合并转发消息（里面有大概80条），bot直接卡死，cpu单核占用100%
过了4分钟后bot恢复正常能处理消息，但是就变成了 [ERROR] apscheduler | Job "_ (trigger: interval[0:01:00], next run at: 2025-01-31 17:43:40 CST)" raised an exception Traceback (most recent call last):
也只有chat_history能响应

猜测：可能是nc默认开启上报解析合并消息导致一瞬间消息量巨大无法处理引发超时

--- a/zhenxun/builtin_plugins/statistics/statistics_hook.py
+++ b/zhenxun/builtin_plugins/statistics/statistics_hook.py
@@ -53,11 +53,7 @@ async def _(
             )
 
 
-@scheduler.scheduled_job(
-    "interval",
-    minutes=1,
-    max_instances=5
-)
+@scheduler.scheduled_job("interval", minutes=1, max_instances=5)
 async def _():
     try:
         call_list = TEMP_LIST.copy()

--- a/zhenxun/builtin_plugins/statistics/statistics_hook.py
+++ b/zhenxun/builtin_plugins/statistics/statistics_hook.py
@@ -56,6 +56,7 @@ async def _(
 @scheduler.scheduled_job(
     "interval",
     minutes=1,
+    max_instances=5
 )
 async def _():
     try:

